spark-bigquery-dsv2/spark-bigquery-dsv2-common/src/main/java/com/google/cloud/spark/bigquery/v2/context/BigQueryDataSourceReaderContext.java

```diff
@@ -455,7 +455,8 @@ public OptionalLong numRows() {
           };
 
       return tableStatisticsContext;
-    } else if (table.getDefinition().getType() == TableDefinition.Type.EXTERNAL) {
+    } else if (table.getDefinition().getType() == TableDefinition.Type.EXTERNAL
```

Comment:
Request a couple of changes:

1. Please populate the number of rows since we have this BLMTs. We had a logic somewhere in the connector to disable number of rows due to lack of column-level statistics. So it is possible that the row count is not used currently, but please add it so it won't be a hidden landmine.  cc @vishalkarve15  to correct me.
2. Since we don't have per-column bytes, let's follow the path for external tables where we report the physical size on disk. The connector will do a rough estimation of the projected size.